registry.k8s.io/images/charts/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS file documentation:
+#  https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md
+
+approvers:
+- tenzen-y
+- mimowo
```

Comment:
@upodroid If we need to add sig labels, please let me know, thanks!

https://github.com/kubernetes/k8s.io/blob/5b37813bd4f4e3e7a675dc26ffcceb4f75691b41/registry.k8s.io/images/k8s-staging-kueue/OWNERS#L17-L18